index.bs

```diff
@@ -240,6 +245,9 @@ interface ImageCapture {
 
   <dt><dfn dict-member for="PhotoSettings"><code>fillLightMode</code></dfn></dt>
   <dd>This reflects the desired <a>fill light mode</a> (flash) setting.</dd>
+
+  <dt><dfn dict-member for="PhotoSettings"><code>imageFormats</code></dfn></dt>
+  <dd>This reflects the desired <a>image format</a> for the captured photo. If the UA supports desired <a>image format</a>, the {{Blob/type!!attribute}} of the {{Blob}} produced by <a>takePhoto()</a> MUST be of the {{imageFormats!!dict-member}} [=image format|type=].</dd>
```

Comment:
>  I would prefer original version. If web developer explicitly requests e.g., "image/tiff", proper type should be returned, or takePhoto() should fail if unsupported type was requested.

Here we're talking about the `PhotoSettings` (not `Constraints`), so what I meant is that the current configured `imageFormat`, if available, should be one of the supported ones.